test/models/CDSAccountStatusSpec.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import utils.SpecBase
+import models.CDSAccountStatus
+import play.api.libs.json.{JsValue, Json, JsString, JsSuccess}
+
+class CDSAccountStatusSpec extends SpecBase {
```

Comment:
Better to rename it to AccountsAndBalancesSpec as CDSAccountStatus is part of that file and tests for all the classes/object should be in the same spec. So if we want to decide to add further tests for the classes/objects present in the class, we can add.